js/index.js

```diff
@@ -17,3 +17,125 @@ for (let i = 0; i < skills.length; i++) {
     skill.innerText = skills[i];
     skillsList.appendChild(skill);
 }
+
+
+const messageForm = document.querySelector('form[name="leave_message"]');
```

Comment:
I found out shorter way: 
const messageForm = document.forms["leave_message"];
Which one is preferred?

Reply:
Ooh I haven't seen this before! If `document.forms` can get you the same item selected, I would maybe use that instead, since it requires less of a  complicated part of the code to live inside a string, where it can be most easily mistyped.